translations/readme/README-de.md

```diff
@@ -0,0 +1,16 @@
+# QCTF School 2018
+
+Dies ist der Aufbewahrungsort für die Entwicklung von QCTF-Schulwettkämpfen 2018 - Vollzeit-CTF-Wettbewerbe für Schulkinder, die von der Mannschaft gehalten werden [Hackerdome](http://hackerdom.ru). Der Wettbewerb wurde am 13. Mai 2018 im [26 Veranstaltungsorte](http://qctf.ru/school-2018/places).
+
+Die Regeln der QCTF-Schule finden Sie auf der offiziellen Website des Wettbewerbs — http://qctf.ru/school-2018/rules
+
+Mehr als 150 Teams nahmen am Wettbewerb teil (jeder - bis zu vier Personen).
+
+Im Aufgabenordner befinden sich Aufgaben. Jede Aufgabe befindet sich in einem eigenen Ordner.
+Die Grundlage für das Verifikationssystem war das System [drape](https://github.com/andgein/drapo), entwickelt von Andrei Geyn.
```

Comment:
Unfortunately I can't check deutsch text but I think it will be better if team and checksystem names will not change: `Hackerdom` and `drapo`

Reply:
ok